src/composables/useIsDarkTheme/index.ts

```diff
@@ -32,9 +34,28 @@ export function useIsDarkThemeElement(el: HTMLElement = document.body): DeepRead
 	return readonly(isDarkTheme)
 }
 
+/**
+ * The real shared composable of the dark theme state.
+ * We need to wrap this to allow to react to injected theme changes.
+ */
+const useInternalIsDarkTheme = createSharedComposable(() => useIsDarkThemeElement())
+
 /**
  * Shared composable to check whether the dark theme is enabled on the page.
+ *
  * Reacts on body data-theme-* attributes change and system theme change.
- * @return {DeepReadonly<Ref<boolean>>} - computed boolean whether the dark theme is enabled
+ * As well as any enforced theme by the `NcThemeProvider`.
+ *
+ * @return Computed boolean whether the dark theme is enabled
  */
-export const useIsDarkTheme = createSharedComposable(() => useIsDarkThemeElement())
+export function useIsDarkTheme(): DeepReadonly<Ref<boolean>> {
+	const isDarkTheme = useInternalIsDarkTheme()
+	const enforcedTheme = inject(INJECTION_KEY_THEME)
```

Comment:
Shouldn't  there be a default value passed? Console is bloating with warnings:

<img width="351" height="46" alt="Image" src="https://github.com/user-attachments/assets/fe7958cd-f2b7-4d53-b81c-d4580276bed6" />